extensions/wikia/UserProfilePageV3/css/UserProfilePage.scss

```diff
@@ -148,6 +148,20 @@ $color-user-profile-gradient-bottom: desaturate(darken($color-menu-highlight, 5%
 				margin: 0 1px 0 6px;
 			}
 		}
+	  	#discussion-bio {
```

Comment:
IIRC one of the reasons for using classes instead of ids in our stylesheets was to make it easier for local wiki admins to apply their own custom CSS (might be worth it to conform this with ComSup, though)